### PR TITLE
עיצוב בסיסי

### LINKED ISSUE #2: עיצוב
צריך לעצב את האתר בעזרת ה design tokens

### PR DESCRIPTION
**Pull request by bennypowers**
## TODO:
- [x] import and use RHDS elements
- [x] use red hat design tokens
- [x] update the footer content (in `_data/footer.yml`) @Liadredhat will do this
- [x] add `rtl`
- [ ] Sidebar nav styles
- [ ] Pagination styles
- [ ] Post list (index) styles
- [ ] stub out a landing page for homepage
- [x] stick footer to bottom on all pages
- [ ] Israel site logo (@ronniemcbride can you whip one up for me in the style of the RHDS logo svg?)
- [ ] RHDS prism theme (lower priority)
- [ ] Hebrew Typography (Noto Sans? font weight and size vs latin?)

### CODE REVIEW

**Comment by bennypowers** [comment]
@heyMP can you take a look at the global footer positioning in the PR preview here? I can kind of fix it with `position: absolute` but that's meh. Am I holding it wrong?

edit: this project is on volunteer basis, so if you don't have time for it that's understandable

**Comment by bennypowers** [comment]
@heyMP fixed it, i had forgotten to specify a row template in the grid settings for the body. https://github.com/RedHat-Israel/red-hat-israel-site/pull/11/commits/25fcd80eb7bb1040f6c75b55eb3eeeedcd807cc7 set's the footer's grid aria row template to max-content, and makes it all better.

File: _data/footer.yaml [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+    content: Facebook
+  - icon: twitter
+    href: https://twitter.com/RedHat
+    content: Twitter
+  - icon: github
+    href: https://github.com/redhat-israel
+    content: GitHub
+
+links:
+  - heading: Products
```
Comment (Liadredhat) [inline _data/footer.yaml]: ניתן להשאיר חלק זה באנגלית
נחליט בהמשך אם זה חיוני כאן

File: _data/footer.yaml [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+    - href: https://www.redhat.com/en/technologies/cloud-computing/openshift
+      content: Red Hat OpenShift
+    - href: https://www.redhat.com/en/technologies/cloud-computing/openshift-data-foundation
+      content: Red Hat OpenShift Data Foundation
+    - href: https://www.redhat.com/en/technologies/linux-platforms/openstack-platform
+      content: Red Hat OpenStack Platform
+    - href: https://www.redhat.com/en/technologies/all-products
+      content: See all products
+  
+  - heading: יצירת קשר
```
Comment (razo7) [inline _data/footer.yaml]: אולי `צור קשר`?

Comment (bennypowers) [reply _data/footer.yaml]: צור קשר לשון זכר

File: _data/footer.yaml [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+    - href: https://www.redhat.com/en/about/social
+      content: עקבו אחרינו בסושיאל
+    
+
+secondary:
+  - heading: רד האט
+    content: |
+      רד האט היא החברה המובילה בעולם לפתרונות תוכנה מבוססי קוד פתוח לארגוני Enterprise. 
+      רד האט מבוססת על כוחה של קהילת הקוד הפתוח בפיתוח ואספקת טכנולוגיות Linux, Cloud, Container, ו-Kubernetes.
+      אנחנו מסייעים ללקוחות לשלב בין יישומי IT חדשים וקיימים, לפתח יישומים מוטי-ענן
```
Comment (razo7) [inline _data/footer.yaml]: `מוטי-ענן` נשמע מוזר.. וחסר נקודה בסוף הפסקה.

File: _data/footer.yaml [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+      content: See all products
+  
+  - heading: יצירת קשר
+    links:
+    - href: https://www.redhat.com/en/contact
+      content: כתבו לנו
+    - href: https://www.redhat.com/en/about/feedback
+      content: שלחו לנו פדיבק
+    - href: https://www.redhat.com/en/about/social
+      content: עקבו אחרינו בסושיאל
```
Comment (razo7) [inline _data/footer.yaml]: אולי `עקבו אחרינו ברשתות החברתיות`?

Comment (razo7) [reply _data/footer.yaml]: או `חפשו אותו ברשתות החברתיות`?

File: _data/footer.yaml (hunk 3 above)
Comment (bennypowers) [reply _data/footer.yaml]: רב-עננים?

File: about/index.md [hunk 5]
```diff
@@ -1,9 +1,9 @@
 ---
 layout: layouts/post.njk
-title: About Me
+title: אודותנו
```
Comment (razo7) [inline about/index.md]: אודותינו

File: about/index.md [hunk 6]
```diff
@@ -1,9 +1,9 @@
...
 templateClass: tmpl-post
 eleventyNavigation:
-  key: About Me
+  key: אודותנו
```
Comment (razo7) [inline about/index.md]: אודותינו

File: _data/footer.yaml [hunk 7]
```diff
@@ -0,0 +1,61 @@
...
+      content: Red Hat OpenStack Platform
+    - href: https://www.redhat.com/en/technologies/all-products
+      content: See all products
+  
+  - heading: צרו קשר
+    links:
+    - href: https://www.redhat.com/en/contact
+      content: כתבו לנו
+    - href: https://www.redhat.com/en/about/feedback
+      content: שלחו לנו פדיבק
```
Comment (razo7) [inline _data/footer.yaml]: פידבק

File: _data/footer.yaml (hunk 3 above)
Comment (razo7) [reply _data/footer.yaml]: נשמע יותר טוב ומספיק טוב לעכשיו :) בטח בהמשך מישהו ימצא צמד מילים טובות יותר.

File: index.njk [hunk 8]
```diff
@@ -1,14 +1,17 @@
...
+  הפוסט האחרון {% else %}
+  {{ maxPosts }} הפוסטים האחרונים{% endif %}
+</h1>
 
 {% set postslist = collections.posts | head(-3) %}
 {% set postslistCounter = collections.posts | length %}
 {% include "postslist.njk" %}
 
-<p>More posts can be found in <a href="{{ '/posts/' | url }}">the archive</a>.</p>
+<p>יש עוד פוסטים ן <a href="{{ '/posts/' | url }}">בארכיו</a>.</p>
```
Comment (razo7) [inline index.njk]: בארכיון

Comment (razo7) [reply index.njk]: להסיר את ה` ן` ב `יש עוד פוסטים ן `

File: _includes/layouts/base.njk [hunk 9]
```diff
@@ -1,47 +1,89 @@
...
+    </aside>
 
+    <main{% if templateClass %} class="{{ templateClass }}"{% endif %}>
       {{ content | safe }}
     </main>
 
-    <footer></footer>
+    <rh-footer>
+      <a slot="logo" href="/en">
+        <img src="{{ '/assets/red-hat-israel-site.svg' | url }}" alt="Red Hat Israel logo" loading="lazy"/>
```
Comment (razo7) [inline _includes/layouts/base.njk:55]: לא רואים את התמונה..

File: _data/footer.yaml [hunk 10]
```diff
@@ -0,0 +1,61 @@
...
+    - href: https://www.redhat.com/en/technologies/cloud-computing/openshift
+      content: Red Hat OpenShift
+    - href: https://www.redhat.com/en/technologies/cloud-computing/openshift-data-foundation
+      content: Red Hat OpenShift Data Foundation
+    - href: https://www.redhat.com/en/technologies/linux-platforms/openstack-platform
+      content: Red Hat OpenStack Platform
+    - href: https://www.redhat.com/en/technologies/all-products
+      content: See all products
+  
+  - heading: צרו קשר
```
Comment (razo7) [inline _data/footer.yaml:34]: זה לא נראה כמו `המוצרים שלנו` (מבחינת style). לא יודע למה

File: _includes/layouts/base.njk (hunk 9 above)
Comment (razo7) [reply _includes/layouts/base.njk:55]: וכאשר לוחצים אליה מגיעים לhttps://redhat-israel.github.io/en. אין לי מושג מה זה.

Comment (bennypowers) [reply _includes/layouts/base.njk:55]: .כי אין כרגע
אפשר בpr הבא

File: _data/footer.yaml (hunk 10 above)
Comment (bennypowers) [reply _data/footer.yaml:34]: נראה לי אותו דבר, וה computed styles זהים

File: index.njk [hunk 11]
```diff
@@ -1,14 +1,17 @@
...
+  הפוסט האחרון {% else %}
+  {{ maxPosts }} הפוסטים האחרונים{% endif %}
+</h1>
 
 {% set postslist = collections.posts | head(-3) %}
 {% set postslistCounter = collections.posts | length %}
 {% include "postslist.njk" %}
 
-<p>More posts can be found in <a href="{{ '/posts/' | url }}">the archive</a>.</p>
+<p>יש עוד פוסטים <a href="{{ '/posts/' | url }}">בארכיו</a>.</p>
```
Comment (razo7) [inline index.njk]: `בארכיו` -> `בארכיון`

File: _data/footer.yaml (hunk 10 above)
Comment (razo7) [reply _data/footer.yaml:34]: In the attached photo [rh-website-view.png](https://postimg.cc/H8gvQJSY) we can see that the font of 
 "כתבו לנו", "שלחו לנו פידבק", ו"עקבו אחרינו ברשתות החברתיות"
looks different than the font of "Red Hat Ansible Automation Platform", "Red Hat Enterprise Linux", "Red Hat OpenShift", "Red Hat OpenShift Data Foundatio", "Red Hat OpenStack Platform", and  "See all products".

By looking at the markdown code it is looks similar, I guess that it uses different style/font for Hebrew and English, and therefore it looks different.

**Review by razo7 — APPROVED** [review]
LGTM

**Comment by ronniemcbride** [comment]
Israel site logo (@ronniemcbride can you whip one up for me in the style of the RHDS logo svg?)
 
 @bennypowers  Sorry I missed this. Do you still need a logo for this?

**Comment by bennypowers** [comment]
Thanks @ronniemcbride, @ShiranHi here at the Israel site offered her expertise, so you're free 🐦

**Comment by ShiranHi** [comment]
@bennypowers It is something we need to ask the Red Hat brand team, I am not sure we can create a new Red Hat logo. As soon as I hear back from them, I'll let you know

**Comment by ShiranHi** [comment]
@bennypowers It seems it is better to use the Red Hat logo as it is... This is the answer I got from the brand team:

_No, we don’t do specific versions of our logo for countries or offices, or use local translations of “Red Hat” in our logo. 
I’d recommend using the usual Red Hat logo in the website header and then using the body of the site to celebrate the work happening in Israel._

**Comment by bennypowers** [comment]
> @bennypowers It seems it is better to use the Red Hat logo as it is... This is the answer I got from the brand team:
> 
> _No, we don’t do specific versions of our logo for countries or offices, or use local translations of “Red Hat” in our logo. I’d recommend using the usual Red Hat logo in the website header and then using the body of the site to celebrate the work happening in Israel._

TYVM @ShiranHi for researching this. I'll open a new issue for some other design related things, and ping you on slack when it's ready

**Pull request merged**